Dockerfile

```diff
@@ -27,14 +29,13 @@ RUN echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen \
 
 ENV LANG=en_US.UTF-8
```

Comment:
I also have the following declared in my Dockerfile, I'm not sure if it would make sense to add them also. I think it generally has the same impact as LANG, but just calling it out in case you wanted to add it.

```bash
ENV LC_ALL=C.UTF-8
ENV PYTHONIOENCODING=UTF-8
```

Reply:
Sounds good, I'll add that.